virttest/qemu_vm.py

```diff
@@ -1977,7 +1977,7 @@ def add_secure_guest_object(params):
                 if not numa_params.get('numa_hmat_lb'):
                     continue
                 nodeid = numa_params['numa_nodeid']
-                initiator = numa_params.get('numa_initiator', nodeid)
+                initiator = numa_params.get('numa_hmat_lb_initiator', nodeid)
```

Comment:
I'm curious about the impact of continuing to use the old naming, and we also need to consider the compatibility of older versions.

Reply:
@PaulYuuu  It need distinguish the `initiator` option for 
1. `-numa node` :  use param `numa_initiator` 
2. `-numa hmat-lb`:  use param `numa_hmat_lb_initiator`

version control is in test case level to decide which option be used or not for different device  :)

Reply:
OK, then I would suggest also updating the cfg.
https://github.com/avocado-framework/avocado-vt/blob/f618e085b9203e9d2210252bedf1664049b83518/virttest/shared/cfg/base.cfg#L253-L255

Reply:
good catch ~

Reply:
@PaulYuuu updated now, and numa hmat-lb block too, thanks! :grin: